.gitlab-ci.yml

```diff
@@ -407,6 +411,21 @@ pre_release_image:
     - export VERSION=`cat version.txt`
     - ./omnibus/package-scripts/publish_image.sh $CI_COMMIT_REF_NAME $STS_DOCKER_TEST_REPO $CI_PROJECT_DIR/outcomes/pkg
 
+pre_release_trace_agent:
+  stage: pre_release
+  image: docker:18.06.0
+  dependencies: 
+    - deps_deb
```

Comment:
you don't need `deps_deb` if you override the before scripts with `before_script: []`

Reply:
removed